.travis.yml

```
@@ -1,20 +1,26 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-docker.sh
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
 script: bash -ex .travis-docker.sh
```

```suggestion
script: travis_wait 50 bash -ex .travis-docker.sh
```
Just in case the solver requires more time (seems to be the case at the moment on alpine for instance (https://travis-ci.org/ocaml/opam-publish/jobs/615657658)